packages/hash/api/src/model/knowledge/entity.model.ts

```diff
@@ -283,13 +251,11 @@ export default class {
   /** @see {@link LinkModel.create} */
   async createOutgoingLink(
     graphApi: GraphApi,
-    params: Omit<LinkModelCreateParams, "sourceEntityModel" | "createdById"> & {
-      createdById?: string;
-    },
+    params: Omit<LinkModelCreateParams, "sourceEntityModel">,
   ): Promise<LinkModel> {
     return await LinkModel.create(graphApi, {
       sourceEntityModel: this,
-      createdById: params.createdById ?? this.accountId,
+      // createdById: params.createdById ?? this.accountId,
```

Comment:
Do we just want to remove this line?

Reply:
Yes, apologies that slipped in. Addressed in https://github.com/hashintel/hash/pull/1081/commits/27352df3e2d3b0833e9106e1b4255515ec2fc64a